### DISCUSSION TITLE
Versionsnummern?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2547] [General]
Müssten nicht alle Versionsnummern auf der aktuellen Version sein ->3.2?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/5fc718ff-7ac6-4b0f-811b-a0d4721251b7)

**Comment by Zefau** [comment]
Ne, das ist ok. Das ist die Versionsnummer an dem es die letzte strukturelle Änderung gab. Also Wenn du von einer früheren Version kommst wird ein Upgrade durchgeführt, d.h. du kannst nicht einfach zurück gehen.

Also der zweite und dritte Einträge ist `devices` und `layout`, oder? Da kannst du einfach zwischen 3.0, 3.1 und 3.2 wechseln, da nichts an der Struktur angepasst wird. Beim ersten und vorletzten Eintrag kannst du nicht mehr einfach von 3.2 zurück gehen, da die Struktur anders ist und die Version vor 3.2 (also 3.0 und 3.1) die Daten dann nicht mehr lesen können.
